Example of token passthrough (#3758)
signalfx/splunk-otel-collector: examples/logs-passthrough-token/README.md

# Splunk HEC Example

This example shows how to use the collector as a passthrough for a HEC token.

The collector receives a request from a local curl command with an authorization token.

> To deploy the example, check out this git repository, open a terminal and in this directory type:

Reviewer: ```suggestion
## Example Deployment
Checkout this git repository, open a terminal and in this directory type:
```

```bash
$> docker-compose up
```

Once running, the application will log the final log record with examples such as:
```
otelcollector  | xxxx-xx-xxTxx:xx:xx.xxx       info    ResourceLog #0
otelcollector  | Resource SchemaURL: 
otelcollector  | Resource attributes:
otelcollector  |      -> host.name: Str(unknown)
otelcollector  |      -> com.splunk.source: Str(foo-0000-0000-0000-0000000000128)
otelcollector  |      -> com.splunk.index: Str(logs)
otelcollector  |      -> com.splunk.hec.access_token: Str(foo-0000-0000-0000-0000000000128)
otelcollector  | ScopeLogs #0
otelcollector  | ScopeLogs SchemaURL: 
otelcollector  | InstrumentationScope  
otelcollector  | LogRecord #0
otelcollector  | ObservedTimestamp: 1970-01-01 00:00:00 +0000 UTC
otelcollector  | Timestamp: 1970-01-01 00:00:00 +0000 UTC
otelcollector  | SeverityText: 
otelcollector  | SeverityNumber: Unspecified(0)
otelcollector  | Body: Str(event)
otelcollector  | Trace ID: 
otelcollector  | Span ID: 
otelcollector  | Flags: 0
otelcollector  |        {"kind": "exporter", "data_type": "logs", "name": "debug"}

> 

Reviewer: ```suggestion
```
